pkg/docker/util.js

```diff
@@ -590,5 +591,63 @@
         return deferred.promise();
     };
 
+    util.find_container_log = function find_container_log(client, entry, resultCallback) {
+        client.call(entry, "org.freedesktop.DBus.Properties", "GetAll",
+                    ["org.freedesktop.Problems2.Entry"]).
+        fail(function (error) {
+            console.log(error);
+        }).
+        done(function (sem_result) {
+            client.call(entry, "org.freedesktop.Problems2.Entry",
+                        "ReadElements", [['container_id'], 0x4]).
+            fail(function (error) {
+                console.log(error);
+            }).
+            done(function (elem_result) {
+                if ('container_id' in elem_result[0]){
+                    var problem_id = sem_result[0]['ID'].v;
+                    // Wait for a while, until ABRT processes the problem and writes it into journal
+                    window.setTimeout(util.problem_log, 2000, problem_id, elem_result[0]['container_id'].v, resultCallback);
```

Comment:
Copying previous comment as that is now hidden as "obsolete": This should poll repeatedly until success for failure, not just once. Processing a large crash can certainly take more than 2s, particularly on a busy system.

Of course it should not try forever, but you can have a retry counter and stop after the 30th iteration (for giving it a whole minute).

Reply:
Ok, so you suggest something like this?
```
setTimeout(2s, 0, callback)

callback(counter):
   log = getJounalLog()
   if not log and counter < 30:
       setTimeout(2s, counter++, callback)
   elif log:
       processLog()
```

Reply:
Right, and perhaps an "else: console.warn(missing report)", for debugging.